src/System Application/App/User Settings/src/UserSettingsImpl.Codeunit.al

```diff
@@ -368,6 +370,40 @@ codeunit 9175 "User Settings Impl."
         ApplicationUserSettings.Modify();
     end;
 
+    procedure LegacyActionBarEnabled(UserSecurityId: Guid): Boolean
+    var
+        ApplicationUserSettings: Record "Application User Settings";
+    begin
+        GetAppSettings(UserSecurityId, ApplicationUserSettings);
+        exit(ApplicationUserSettings."Legacy Action Bar");
+    end;
+
+    procedure DisableLegacyActionBar(UserSecurityId: Guid)
+    var
+        ApplicationUserSettings: Record "Application User Settings";
+    begin
+        GetAppSettings(UserSecurityId, ApplicationUserSettings);
+        ApplicationUserSettings."Legacy Action Bar" := false;
+        ApplicationUserSettings.Modify();
+    end;
+
+    procedure EnableLegacyActionBar(UserSecurityId: Guid)
+    var
+        ApplicationUserSettings: Record "Application User Settings";
+        CustomerExperienceSurvey: Codeunit "Customer Experience Survey";
+        FormsProId: Text;
```

Comment:
FormsProcessId? or.. I'm not sure what Pro stands for, would be great to have written out.

Reply:
I do not know what they mean either. This is old code that I was informed to take and place in directly to show the survey.

Reply:
Also why I didn't change the literal text names from 'modernactionbar'... to 'legacyactionbar'. I don't know how that breaks stuff.